### ISSUE TITLE
Error ListView within CarouselSampleEmbeddedView

### ISSUE THREAD

**Issue by AlfredoSolisFrausto** [AndreiMisiukevich/CardView#66]
Hi again, I found a problem in CarouselSampleEmbeddedView.xaml, if you insert a ListView in a layout of a TabItem.Content (I tried with a StackLayout), you change of TabItem in the execution and then return to the TabItem that has the ListView giving a tab on the header text, the ItemTemplate's ListView disappears.  When do you drag in the TabItem.Content, doesn't happen.

**Comment by AndreiMisiukevich** [comment]
Hi
It's difficult for understanding

Could you please fork this repo, modify the sample, then I will debug this case

**Comment by AlfredoSolisFrausto** [comment]
That's right, I pushed a commit in the fork to reproduce the error found in CarouselSampleEmbeddedView.xaml. I also include a reference image: https://drive.google.com/open?id=1szRW4xD1laGSOXe2jELtq_T-6eInfhXS

Note: In android ocurrs the same.

**Comment by AndreiMisiukevich** [comment]
very well)
--
If somebody wants to investigate this issue, i will be grateful =)
Or i will look through it on weekend

**Comment by AlfredoSolisFrausto** [comment]
Excellent @AndreiMisiukevich now it works correctly!
